src/lib/LUA/tx_devLUA.cpp

```diff
@@ -459,6 +515,26 @@ static void luahandSimpleSendCmd(struct luaPropertiesCommon *item, uint8_t arg)
     {
       RxWiFiReadyToSend = true;
     }
+  #if defined(RADIO_SX127X)
+    else if ((void *)item == (void *)&luaStartStopBeacon)
+    {
+      if (BeaconEnabled) 
+      {
+        #if (defined(PLATFORM_ESP32) || defined(PLATFORM_ESP8266))
+        msg = "Deactivating...";
+        rebootTime = millis() + 400;
```

Comment:
You don't need the #if and #else part as we're removing STM32 in the master branch.

Reply:
It is fully removed, we're just cleaning up some of the left over ifdef's